install.sh

```diff
@@ -224,7 +224,14 @@ setup_env() {
     if [ -n "${INSTALL_K3S_BIN_DIR}" ]; then
         BIN_DIR=${INSTALL_K3S_BIN_DIR}
     else
+        # --- use /usr/local/bin if we can right to it, otherwise try /opt
         BIN_DIR=/usr/local/bin
+        touch ${BIN_DIR}/k3s-ro-test 2>/dev/null ||:
```

Comment:
These 6 lines (229–234) can be simplified to a oneliner:

`touch ${BIN_DIR}/k3s-ro-test 2>/dev/null && rm -rf ${BIN_DIR}/k3s-ro-test || BIN_DIR=/opt/bin`